sdk/test-utils/perfstress/src/program.ts

```diff
@@ -67,7 +67,11 @@ export class PerfStressProgram {
 
   private getOperationsPerSecond(parallels: PerfStressParallel[]): number {
     return parallels.reduce((sum, parallel) => {
-      return sum + parallel.completedOperations / (parallel.lastMillisecondsElapsed / 1000);
+      let parallelResult = 0;
+      if (parallel.lastMillisecondsElapsed > 0) {
```

Comment:
```suggestion
      if (parallel.completedOperations > 0) {
```

I believe `completedOperations` will also be `0` when `lastMillisecondsElapsed` is `0`, and I think it's clearer to check `completedOperations`.

Reply:
Updated